src/core/components/InstallButton/index.js

```diff
@@ -269,6 +270,10 @@ export class InstallButtonBase extends React.Component {
       } : (event) => {
         this.installExtension({ event, installURL });
       };
+
+      const buttonText = ADDON_TYPE_THEMES.includes(addon.type)
+        ? 'Install Theme' : 'Add to Firefox';
```

Comment:
I don't think this will work because the extraction mechanism does not understand variables. We have to use `gettext('blah')` to have `blah` extracted.

So that's fine to have a `buttonText` variable here, but it needs to call `gettext` twice.

Reply:
okay.. I wasn't sure about the variable with that way i did it here but makes sense